scripts/jenkins/build_docker.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -euxo pipefail
+
+if ! which docker
```

Comment:
NIT https://stackoverflow.com/questions/37056192/which-vs-command-v-in-bash

```suggestion
if ! command -v docker
```